game/scripts/vscripts/GameMode.ts

```diff
@@ -37,8 +37,7 @@ export class GameMode {
     context: TutorialContext = {};
 
     public static Precache(this: void, context: CScriptPrecacheContext) {
-        // PrecacheResource("particle", "particles/units/heroes/hero_meepo/meepo_earthbind_projectile_fx.vpcf", context);
-        // PrecacheResource("soundfile", "soundevents/game_sounds_heroes/game_sounds_meepo.vsndevts", context);
+        PrecacheResource("soundfile", "soundevents/tutorial_dialogs.vsndevts", context);
```

Comment:
I suggest splitting the precache resources per chapter. People with too weak PCs might have trouble precaching all of those at once.